src/SWP/Bundle/CoreBundle/Tests/Twig/InlineAdsTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Tests\Twig;
+
+use SWP\Bundle\FixturesBundle\WebTestCase;
+
+class InlineAdsTest extends WebTestCase
+{
+    /**
+     * @var \Twig_Environment
+     */
+    private $twig;
+
+    public function setUp()
```

Comment:
please add `:void` return type to all the functions in this class and `declare(strict_types=1);` after `<?php`